.github/workflows/vite.yaml

```
@@ -0,0 +1,41 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: client build and test
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  admin-build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
```

```suggestion
jobs:
  admin-build:
    runs-on: ubuntu-latest
    strategy:
```

unnecessary empty lines, should fix whole file